packages/vite/src/node/build.ts

```diff
@@ -736,6 +736,7 @@ async function buildEnvironment(
       const watcher = watch({
         ...rollupOptions,
         watch: {
+          ...rollupOptions.watch,
           ...options.watch,
           chokidar: resolvedChokidarOptions,
```

Comment:
Should we also handle for `chokidar` options too? I think we can spread from rollupOptions on Line 729 here: https://github.com/vitejs/vite/blob/e899bc7c73a27cdf327875e5d696c50d396a7fc2/packages/vite/src/node/build.ts#L728-L733